compose/config/fields_schema.json

```diff
@@ -2,15 +2,18 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
 
   "type": "object",
+  "id": "fields_schema.json",
```

Comment:
I'm not too fond of having the filename being repeated here, especially because this object describes the root schema (dictionary of services).

Alternatives: `services`, `compose_services`, `services_list` ?